WORKSPACE

```diff
@@ -1,18 +1,23 @@
+workspace(name = "angular_devkit_src")
```

Comment:
the `_src` suffix in angular/angular was because I had two workspaces in that repo, but I'm trying to move away from that. Just `angular_devkit` is better.
Note that the workspace name will appear in absolute imports, just as if you were deep-importing from under an npm package. That means that code is more portable: a file could be copied from inside the project to a dependent project and imports should stay the same.